src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/components/structure/page/customheaderlibs.html

```diff
@@ -18,3 +18,20 @@
 
 <!--/* Include Context Hub */-->
 <sly data-sly-resource="${symbol_dollar}{'contexthub' @ resourceType='granite/contexthub/components/contexthub'}"/>
+<!--/* Data layer JavaScript object: */-->
```

Comment:
I would rather we add the datalayer to the Core Components instead, so everyone can benefit from it.